### PR TITLE
touying-brandred-uobristol:0.1.0

### PR DESCRIPTION
**Pull request by HPDell**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: A template to create Touying slides following design of University of Bristol.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by OrangeX4** [comment]
Based on the discussion in #908, the name will need to incorporate some creative component like `simpl` in `touying-simpl-hkustgz`. Thanks.

**Comment by HPDell** [comment]
What does the `simpl` mean? Any more examples for the creative component? I haven’t fully understand what name I need here.

**Comment by OrangeX4** [comment]
> Additional guidance for template packages: It is often desirable for template names to feature the name of the organization or publication the template is intended for. However, it is still important to us to accommodate multiple templates for the same purpose. Hence, template names shall consist of a unique, non-descriptive part followed by a descriptive part. For example, a template package for the fictitious American Journal of Proceedings (AJP) could be called organized-ajp or eternal-ajp. Package names should be short and use the official entity abbreviation. Template authors are encouraged to add the full name of the affiliated entity as a keyword.

**Comment by HPDell** [comment]
Hi,

I've renamed the package to `touying-brandred-uobristol`. Is it a valid name?

**Comment by OrangeX4** [comment]
I think it's okay, nice name.

File: packages/preview/touying-brandred-uobristol/0.1.0/typst.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+[package]
+name = "touying-brandred-uobristol"
+version = "0.1.0"
+entrypoint = "lib.typ"
+authors = ["HPDell <huyg0180110559@outlook.com>"]
+license = "MIT"
+description = "Touying Slide Theme for University of Bristol"
+homepage = "https://github.com/HPDell/touying-uobristol"
```
Comment (laurmaedje) [inline packages/preview/touying-brandred-uobristol/0.1.0/typst.toml:8]: Just repository suffices if they are the same.
```suggestion
```

**Review by laurmaedje — COMMENTED** [review]
The folder with the old name is still there. It should be deleted.

File: packages/preview/touying-brandred-uobristol/0.1.0/typst.toml (hunk 1 above)
Comment (HPDell) [reply packages/preview/touying-brandred-uobristol/0.1.0/typst.toml]: This issue has been fixed.

**Comment by laurmaedje** [comment]
Thanks!

**Pull request merged**